node-src/lib/findChangedDependencies.ts

```diff
@@ -60,29 +60,44 @@ export const findChangedDependencies = async (ctx: Context) => {
 
   ctx.log.debug({ pathPairs }, `Found ${pathPairs.length} manifest/lockfile pairs to check`);
 
-  const tracedPairs = pathPairs.filter(([manifestPath, lockfilePath]) => {
-    if (untraced.some((glob) => matchesFile(glob, manifestPath))) return false;
-    if (untraced.some((glob) => matchesFile(glob, lockfilePath))) return false;
-    return true;
-  });
-  const untracedCount = pathPairs.length - tracedPairs.length;
-  if (untracedCount) {
-    ctx.log.debug(`Skipping ${untracedCount} manifest/lockfile pairs due to --untraced`);
+  // Now filter out any pairs that don't have git changes, or for which the manifest is untraced
+  const filteredPathPairs = pathPairs
+    .map(([manifestPath, lockfilePath]) => {
+      const commits = packageMetadataChanges
+        .filter(
+          ({ changedFiles }) =>
+            changedFiles.includes(manifestPath) || changedFiles.includes(lockfilePath)
```

Comment:
A single `some` operation would be more efficient. The lockfile is more likely to have changed than the manifest file.